pgmax/fg/graph.py

```diff
@@ -790,11 +790,13 @@ def BP(bp_state: BPState, num_iters: int) -> Tuple[Callable, Callable, Callable]
         num_iters: Number of belief propagation iterations.
 
     Returns:
-        run_bp: Function for running belief propagation for num_iters.
+        tuple containing
+
+            run_bp(callable): Function for running belief propagation for num_iters.
```

Comment:
I think we don't need these additional `callable` annotations (to be consistent with internal guidelines).